.github/workflows/run_opa_tests.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: Setup OPA
       uses: open-policy-agent/setup-opa@v2
       with:
-        version: <0.50
+        version: latest
```

Comment:
We might consider using our bespoke scripts since there is/will be an expectation of which version and where the OPA agent is placed.